packages/nextcloud/test/provisioning_api_test.dart

```diff
@@ -14,24 +14,42 @@ void main() {
     late TestNextcloudClient client;
     setUp(() async {
       container = await getDockerContainer(image);
-      client = await getTestClient(container);
+      client = await getTestClient(
+        container,
+        username: 'admin',
+      );
     });
     tearDown(() => container.destroy());
 
-    test('Get current user', () async {
-      final user = await client.provisioningApi.getCurrentUser();
-      expect(user.ocs.data.id, 'user1');
-      expect(user.ocs.data.displayname, 'User One');
-      expect(user.ocs.data.displaynameScope, 'v2-federated');
-      expect(user.ocs.data.language, 'en');
+    group('Users', () {
+      test('Get current user', () async {
+        final user = await client.provisioningApi.users.getCurrentUser();
+        expect(user.ocs.data.id, 'admin');
+        expect(user.ocs.data.displayName, 'admin');
+        expect(user.ocs.data.displaynameScope, 'v2-federated');
+        expect(user.ocs.data.language, 'en');
+      });
+
+      test('Get user by username', () async {
+        final user = await client.provisioningApi.users.getUser(userId: 'user1');
+        expect(user.ocs.data.id, 'user1');
+        expect(user.ocs.data.displayname, 'User One');
+        expect(user.ocs.data.displaynameScope, null);
+        expect(user.ocs.data.language, 'en');
+      });
     });
 
-    test('Get user by username', () async {
-      final user = await client.provisioningApi.getUser(userId: 'user1');
-      expect(user.ocs.data.id, 'user1');
-      expect(user.ocs.data.displayname, 'User One');
-      expect(user.ocs.data.displaynameScope, 'v2-federated');
-      expect(user.ocs.data.language, 'en');
+    group('Apps', () {
+      test('Get apps', () async {
+        final apps = await client.provisioningApi.apps.getApps();
+        expect(apps.ocs.data.apps, hasLength(39));
+      });
+
+      test('Get app infos', () async {
+        for (final app in (await client.provisioningApi.apps.getApps()).ocs.data.apps) {
+          await client.provisioningApi.apps.getAppInfo(app: app);
+        }
+      });
```

Comment:
I'm not a fan of tests without any expect (I can see why it's done though)

But I really dislike `(await client.provisioningApi.apps.getApps()).ocs.data.apps`
The result of getApps should be in a separate variable raising thje question why this isn't merged with the test above it.